packages/hmr-plugin/src/tests/hmr-mock.ts

```diff
@@ -82,17 +81,10 @@ export class AppMockModuleNoHmrLifeCycle {
 }
 
 function createRootNode(selector = 'app-root'): void {
-  const document = TestBed.get(DOCUMENT);
-  const adapter: DomAdapter = new BrowserDomAdapter();
-
-  const root = adapter.firstChild(
-    adapter.content(adapter.createTemplate(`<${selector}></${selector}>`))
-  );
-
-  const oldRoots = adapter.querySelectorAll(document, selector);
-  oldRoots.forEach(oldRoot => adapter.remove(oldRoot));
```

Comment:
So, we don't need to remove the old roots anymore?

Reply:
Yes, now Angular itself destroys the root.